atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -212,11 +221,25 @@ public static SerializableTransactionManager create(
             LockServerOptions lockServerOptions,
             boolean allowHiddenTableAccess,
             String userAgent) {
+        TransactionManagerOptions options = TransactionManagerOptions.builder()
+                .allowHiddenTableAccess(allowHiddenTableAccess)
+                .config(config)
+                .env(env)
+                .lockServerOptions(lockServerOptions)
+                .runtimeConfigSupplier(optionalRuntimeConfigSupplier)
+                .schemas(schemas)
+                .userAgent(userAgent)
+                .build();
+        return create(options);
+    }
+
+    public static SerializableTransactionManager create(TransactionManagerOptions options) {
```

Comment:
I think my ideal API for this would be
```
TransactionManager txnManager = TransactionManagers.builder()
.schemas(...)
.otherStuff()
.build();
```

Reply:
should the api really be `TransactionManager.builder()` then? Note specifically the lack of `s`?

Reply:
I think we probably keep it in `TransactionManagers` as `TransactionManager` is the base interface, but most things actually deal with `LockAwareTransactionManager`.

It'd be nice if the actual method to create was `TransactionManagers.buildSerializable()` for `SerializableTransactionManager` (and if we actually needed/wanted  `TransactionManagers.buildSnapshot()` for returning a`SnapshotTransactionManager`)